inst/@sym/eye.m

```diff
@@ -0,0 +1,86 @@
+%% Copyright (C) 2016 Lagu
+%%
+%% This file is part of OctSymPy.
+%%
+%% OctSymPy is free software; you can redistribute it and/or modify
+%% it under the terms of the GNU General Public License as published
+%% by the Free Software Foundation; either version 3 of the License,
+%% or (at your option) any later version.
+%%
+%% This software is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty
+%% of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+%% the GNU General Public License for more details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @documentencoding UTF-8
+%% @defmethod @@sym eye (@var{n})
+%% @defmethodx @@sym eye (@var{n}, @var{m})
+%% @defmethodx @@sym eye (@var{n}, @var{m}, @var{class})
+%% Return an identity matrix.
+%%
+%% Example:
+%% @example
+%% @group
+%% y = eye (sym(3))
+%%   @result{} y = (sym 3×3 matrix)
+%%  ⎡1  0  0⎤
+%%  ⎢       ⎥
+%%  ⎢0  1  0⎥
+%%  ⎢       ⎥
+%%  ⎣0  0  1⎦
+%% @end group
+%% @end example
+%%
+%% @seealso{@@sym/zeros, @@sym/ones}
+%% @end defmethod
+
+%% Source: http://docs.sympy.org/dev/modules/matrices/matrices.html
```

Comment:
No, its because when i read or try to works with this function i think its very useful know where we can find exactly the functions in Sympy, instead of start searching in all the docs, because are a lot of it and with differents informations, this link its the full doc of that function.